.github/workflows/pull-request.yaml

```diff
@@ -34,10 +34,10 @@ jobs:
     secrets: inherit
 
   # Integration Tests
-  bats:
-    needs: build
-    secrets: inherit
-    uses: ./.github/workflows/bats.yaml
+  # bats:
+  #   needs: build
+  #   secrets: inherit
+  #   uses: ./.github/workflows/bats.yaml
```

Comment:
We may want to do this in another PR for easier tracking. Maybe hold on merging this until we make an effort to remove.

Reply:
https://github.com/posit-dev/publisher/pull/2558 was opened to remove these so we can solve the conflict with whatever merges first.